sessiondescription_test.go

```diff
@@ -0,0 +1,12 @@
+package sdp
+
+const (
+	exampleAttrExtmap1     = "extmap:1 http://example.com/082005/ext.htm#ttime"
+	exampleAttrExtmap1Line = attributeKey + exampleAttrExtmap1 + endline
+	exampleAttrExtmap2     = "extmap:2/sendrecv http://example.com/082005/ext.htm#xmeta short"
+	exampleAttrExtmap2Line = attributeKey + exampleAttrExtmap2 + endline
+	exampleAttrExtmap3     = "extmap:257/sendrecv http://example.com/082005/ext.htm#xmeta short"
```

Comment:
Super minor change, can we rename these so it is clear which examples are of failing syntax?

Reply:
yeah, good idea